src/js/components/CheckBox/StyledCheckBox.js

```diff
@@ -12,8 +12,14 @@ const hoverStyle = css`
   :hover input:not([disabled]) + div,
   :hover input:not([disabled]) + span {
     border-color: ${props =>
+      props.theme.checkBox.hover.border.color &&
       normalizeColor(props.theme.checkBox.hover.border.color, props.theme)};
   }
+  :hover {
```

Comment:
Why does the `background` have different hover conditions than `border`? 
the conditions for the border are on lines 12-13. I would think they should all be placed together.

Reply:
The conditions for the `hover.border.color` are not applied to the whole box container just the checkbox 

The `hover.background.color` is being applied to the whole box containing both label and checkbox.